package.json

```diff
@@ -99,11 +100,11 @@
   },
   "peerDependencies": {
     "@tanstack/react-query": ">=5.0.0",
-    "@wagmi/core": "beta",
+    "@wagmi/core": "^2.6.5",
     "op-viem": "1.3.1-alpha",
     "typescript": ">=5.0.4",
-    "viem": "beta",
-    "wagmi": "beta"
+    "viem": "^2.7.9",
+    "wagmi": "^2.5.7"
```

Comment:
i think these be `2.x`? that's what we have in op-viem and how wagmi has their peer dependencies set up

Reply:
I can change it to that, my understanding of the `^` operator is that `^2.7.9` and `^2.x` have the same effect as they both allow any minor and patch version within 2